python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -117,10 +183,28 @@ def write_catalog(self, obsHistID, out_dir='.', fov=2):
         knots_name = 'knots_cat_%d.txt' % obsHistID
         #agn_name = 'agn_cat_%d.txt' % obshistid
 
+	# SN Data
+        snDataDir = os.path.join(getPackageDir('sims_GCRCatSimInterface'), 'data')
+        sncsv_hostless_uDDF = 'uDDF_hostlessSN_trimmed.csv'
+        sncsv_hostless_pDC2 = 'MainSurvey_hostlessSN_trimmed.csv'
+        sncsv_hostless_pDC2hz = 'MainSurvey_hostlessSN_highz_trimmed.csv'
+        sncsv_hosted_uDDF = 'uDDFHostedSNPositions_trimmed.csv'
+        sncsv_hosted_pDC2 = 'MainSurveyHostedSNPositions_trimmed.csv'
+
+        snpopcsvs = list(os.path.join(snDataDir, n) for n in 
+                        [sncsv_hostless_uDDF,
+                         sncsv_hostless_pDC2,
+                         sncsv_hostless_pDC2hz,
+                         sncsv_hosted_uDDF,
+                         sncsv_hosted_pDC2])
+
+        names = list(snpop.split('/')[-1].split('.')[0].strip('_trimmed')
+                         for snpop in snpopcsvs)
```

Comment:
We should build the `object_catalogs` more carefully.  Let's insert the following here:
```
        object_catalogs = [star_name, gal_name] + [x + '.txt'  for x in names[:5]]
```

Reply:
This is better, but I should probably drop the `5` in case the size of the list is changed. It is the complete list.